packages/forklift-console-plugin/src/modules/Plans/views/create/PlanCreatePage.tsx

```diff
@@ -59,65 +75,68 @@ export const PlanCreatePage: React.FC<{ namespace: string }> = ({ namespace }) =
   });
   useSaveEffect(state, dispatch);
 
-  const steps = [
-    {
-      id: 'step-1',
-      name: 'Select source provider',
-      component: (
-        <SelectSourceProvider
-          projectName={projectName}
-          filterState={filterState}
-          filterDispatch={filterDispatch}
-          dispatch={dispatch}
-          state={state}
-          providers={providers}
-          selectedProvider={selectedProvider}
-        />
-      ),
-      enableNext: filterState?.selectedVMs?.length > 0,
-    },
-    {
-      id: 'step-2',
-      name: 'Create migration plan',
-      component: (
-        <ProvidersCreateVmMigrationPage
-          state={state}
-          dispatch={dispatch}
-          emptyContext={emptyContext}
-        />
-      ),
-      enableNext:
-        !emptyContext &&
-        !(
-          !!state?.flow?.apiError ||
-          Object.values(state?.validation || []).some((validation) => validation === 'error') ||
-          state?.validation?.planName === 'default'
-        ),
-      canJumpTo: filterState?.selectedVMs?.length > 0,
-      nextButtonText: 'Create migration plan',
-    },
-  ];
+  const anyValidationError = Object.values(state?.validation || []).some(
+    (validation) => validation === 'error',
+  );
+
+  const planNameValidationDefault = state?.validation?.planName === 'default';
 
   const title = 'Plans wizard';
+
   return (
     <>
       <PageSection variant="light">
         <Title headingLevel="h2">{'Create migration plan'}</Title>
       </PageSection>
-
-      <PageSection variant="light">
+      <PageSection variant="light" className="forklift--create-plan--wizard-container">
         <Wizard
-          className="forklift--create-plan--wizard-appearance-order"
-          navAriaLabel={`${title} steps`}
-          mainAriaLabel={`${title} content`}
-          steps={steps}
+          className="forklift--create-plan--wizard-content"
+          shouldFocusContent
+          title={title}
+          startIndex={startAtStep}
+          onClose={() =>
+            window.location.replace(createPlanFromPlansList ? plansListURL : `${providerURL}/vms`)
```

Comment:
Why touch the direct window location and not use navigate from the useNavigate hook?

Reply:
The navigate hook is supported by React Router v6: https://reactrouter.com/en/v6.3.0/upgrading/v5#use-usenavigate-instead-of-usehistory. 
The kubevirt plugin and console code are still using `react-router@5.3.4` with `react-router-dom-v5-compat` and no usage found for the `useNavigate`.
I tried using `react-router-dom-v5-compat` here as wel,l just to be aligned, but it doesn't help of course and I'm not sure we can upgrade to 6 before the console does. 
Are you aware of a plugin using the useNavigate hook?